tests/framework/PostProcessors/DataMiningPostProcessor/Clustering/gold/time_dep_clustering_filtered/print0_0.csv

```diff
@@ -1,101 +1,101 @@
--14.0656046487,-2.73817626804,33.110284471,0.205
--13.7850200056,-3.87091910611,32.6124846003,0.21
--13.4686242579,-4.86232919606,32.2764253174,0.215
--13.1260041386,-5.72295870224,32.0706094919,0.22
--12.761784797,-6.46326324587,31.9665890349,0.225
--12.3777958579,-7.09311540099,31.9389744069,0.23
--11.974621899,-7.62158344668,31.965239767,0.235
--11.5526616223,-8.05688729191,32.0254891737,0.24
--11.1128058804,-8.40646472494,32.1022610552,0.245
--10.6568226931,-8.67709884049,32.1803948667,0.25
--10.1875174716,-8.87507122575,32.2469540419,0.255
--9.70872210076,-9.00631585033,32.2911846992,0.26
--9.22515723202,-9.07655647538,32.304484618,0.265
--8.74220668238,-9.09141655104,32.2803583009,0.27
--8.26563941254,-9.05649556417,32.2143391715,0.275
--7.80131157829,-8.97740994901,32.1038673937,0.28
--7.35487746281,-8.85980011194,31.9481199849,0.285
--6.93153314934,-8.70930784703,31.7477975604,0.29
--6.53581059083,-8.53153037726,31.5048781859,0.295
--6.17143273781,-8.33195839862,31.2223527669,0.3
--5.84123331843,-8.11590583253,30.9039579014,0.305
--5.54713849655,-7.88843858112,30.553921353,0.31
--5.29020259424,-7.65430857267,30.1767327302,0.315
--5.07068672833,-7.41789797482,29.7769482881,0.32
--4.88816764997,-7.18317685018,29.3590347019,0.325
--4.74166410738,-6.95367593015,28.9272528369,0.33
--4.62976930898,-6.73247474788,28.4855793836,0.335
--4.55078010651,-6.52220420399,28.037661983,0.34
--4.502815914,-6.32506179424,27.5868021682,0.345
--4.48392277306,-6.14283720622,27.1359599994,0.35
--4.49216011596,-5.9769457629,26.6877744758,0.355
--4.52566952267,-5.82846719821,26.2445944635,0.36
--4.58272606333,-5.69818743065,25.8085157748,0.365
--4.66177368136,-5.58664129392,25.381421008,0.37
--4.76144655983,-5.49415453266,24.9650196879,0.375
--4.88057860261,-5.42088373538,24.5608870616,0.38
--5.01820313051,-5.3668532221,24.1705005649,0.385
--5.17354471302,-5.33198821294,23.7952734796,0.39
--5.34600478835,-5.31614386295,23.4365856478,0.395
--5.53514241249,-5.31912995549,23.095811336,0.4
--5.74065115667,-5.34073120119,22.7743444518,0.405
--5.96233286143,-5.38072319674,22.4736213472,0.41
--6.20006867092,-5.43888416321,22.1951414053,0.415
--6.45378751901,-5.51500261398,21.9404855209,0.42
--6.72343202574,-5.60888110448,21.7113324574,0.425
--7.008921591,-5.72033619661,21.5094729003,0.43
--7.31011234613,-5.84919473605,21.3368208351,0.435
--7.62675354764,-5.99528649706,21.1954216527,0.44
--7.958439979,-6.15843320211,21.0874561342,0.445
--8.30455997575,-6.3384338798,21.0152391807,0.45
--8.66423882218,-6.5350464894,20.9812118456,0.455
--9.03627750285,-6.74796572268,20.987924898,0.46
--9.41908715215,-6.97679690069,21.0380118093,0.465
--9.81062005694,-7.22102592584,21.1341487415,0.47
--10.2082987558,-7.47998533895,21.2789988595,0.475
--10.6089456682,-7.75281668063,21.4751381402,0.48
--11.0087167901,-8.03842957938,21.7249598992,0.485
--11.4030443018,-8.33545830046,22.030555582,0.49
--11.786594409,-8.64221690059,22.3935701026,0.495
+time,x,y,z
```

Comment:
is there a reason why this historyset is different?

Reply:
nvm: the history set with this label have changed during the clustering

Reply:
yes, the label are changed during clustering